### PR TITLE
Add setting attribute for fileio

### PR DESCRIPTION
**Pull request by qingwangrh**
Using lun_attrs to define attributes of fileio lun
ID:2007146
Signed-off-by: qingwangrh <qinwang@redhat.com>

### CODE REVIEW

**Comment by qingwangrh** [comment]
Some case need to emulate disk with block_size 4096, it need to set attribute for the  backend fileio.

**Comment by qingwangrh** [comment]
@YongxueHong @XueqiangWei Please help to review,thanks

**Comment by XueqiangWei** [comment]
@qingwangrh Please add the ID number. Thanks.

**Comment by qingwangrh** [comment]
> @qingwangrh Please add the ID number. Thanks.

updated

**Comment by zhencliu** [comment]
LGTM

File: virttest/iscsi.py [hunk 1]
```diff
@@ -208,6 +208,7 @@ def __init__(self, params, root_dir):
         self.target = params.get("target")
         self.export_flag = False
         self.luns = None
+        self.lun_attrs = params.get("lun_attrs")
```
Comment (luckyh) [inline virttest/iscsi.py:211]: Suggest using `iscsi_lun_attrs` as the param name.

Comment (qingwangrh) [reply virttest/iscsi.py]: updated

**Comment by XueqiangWei** [comment]
LGTM,
Acked-by: xuwei@redhat.com

**Review by luckyh — APPROVED** [review]

**Pull request merged**